gulpfile.js

```diff
@@ -0,0 +1,54 @@
+const { task, src, series, dest } = require('gulp');
+const log = require('fancy-log');
+const modifyFile = require('gulp-modify-file');
+const prettier = require('gulp-prettier');
+
+task('check-wiki.json', done => {
+  src('./wiki.json')
+    .pipe(
+      modifyFile(content => {
+        let hasWrongFormatName = false;
+        const wikiData = JSON.parse(content);
+
+        for (let idx = 0; idx < wikiData.length; idx++) {
+          if (isValidName(wikiData[idx].name) === false) {
+            hasWrongFormatName = true;
+            break;
+          }
+        }
```

Comment:
```suggestion
      modifyFile(wikiData => {
        const records = JSON.parse(wikiData);

        const hasWrongFormatName = records.some(record => !isValidName(record.name));
```

1. [Array.prototype.some](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/some) 사용해봐도 좋을 것 같슴다 ㅎㅎ
2. 배열 변수 네이밍은 복수형으로해봐도 조흘듯합니다..!